src/main/java/de/photon/anticheataddition/commands/subcommands/internaltestcommands/TestInventoryClick.java

```diff
@@ -38,6 +39,11 @@ protected void execute(CommandSender sender, Queue<String> arguments)
         final var user = parseUser(sender, arguments.poll());
         if (user == null) return;
 
+        if (ServerVersion.ACTIVE == ServerVersion.MC120) {
```

Comment:
what about 1.21?

Reply:
It works for 1.21, this is due to a change in the InventoryView interface, which I have updated to compile with the latest version. Pre-1.19 versions have a different approach, so it is only non-functional on 1.20.